lib/commands/recordscreen.js

```diff
@@ -239,17 +273,17 @@ export default {
 
     let {mjpegServerScreenshotQuality, mjpegServerFramerate} =
       /** @type {import('appium-webdriveragent').WDASettings} */ (
-        await this.proxyCommand('/appium/settings', 'GET')
-      );
+      await this.proxyCommand('/appium/settings', 'GET')
+    );
```

Comment:
```suggestion
        await this.proxyCommand('/appium/settings', 'GET')
      );
```
(just to keep the indentation)